internal/lookout/ui/src/services/ComputeResourcesService.ts

```diff
@@ -0,0 +1,111 @@
+import { load } from "js-yaml"
+
+// A simple interface that our PodSpec has.
+export interface JobYaml {
+  podSpec: {
+    containers: Container[]
+  }
+}
+
+export interface ContainerInfo {
+  command?: string[]
+  arguments?: string[]
+  cpu?: string[]
+  memory?: string[]
+  gpu?: string[]
+  "ephermal-storage"?: string[]
```

Comment:
```suggestion
  "ephemeral-storage"?: string[]
```
Please test this to make sure it works